pffft/LICENSE

```diff
@@ -0,0 +1,45 @@
+Copyright (c) 2013  Julien Pommier ( pommier@modartt.com )
```

Comment:
Could you move this into the `third-party` directory (or switch to system fftw3 :grin:)?